pwa/src/services/validateLength.ts

```diff
@@ -0,0 +1,7 @@
+export const validateLength = (data: string): boolean | string => {
+  const invalidMessage = "This field contains more than the 255 character limit";
+
+  if (!data) return true; // empty data should not be validated
+
+  if (data.length > 255) return invalidMessage;
+};
```

Comment:
If you implement the `maxLength` as mentioned in the other comment, you won't need this anymore.